app/Bookstore.Web/appsettings.json

```diff
@@ -3,18 +3,21 @@
     "BookstoreDatabaseConnection": "Data Source=bookstore.db"
   },
   "Logging": {
-    "LogGroup": "BobsBookstore",
+    "LogGroup": "BobsUsedBookStore",
+    "Region": "us-east-1",
```

Comment:
I think we can remove the region from the config. The logger will fall back to using the profile's region during development.

Reply:
Done